Patch created by Jasper Grimm (UoY)
Fix Rmath include statements to match installed include hierarchy
diff -Nru vt-0.57721/consolidate.h vt-0.57721b/consolidate.h
--- vt-0.57721/consolidate.h	2021-05-14 16:42:01.000000000 +0100
+++ vt-0.57721b/consolidate.h	2021-05-17 14:40:08.000000000 +0100
@@ -25,7 +25,7 @@
 #define CONSOLIDATE_VARIANTS_H
 
 #include "program.h"
-#include "Rmath/Rmath.h"
+#include "Rmath.h"
 #include "log_tool.h"
 
 void consolidate(int argc, char **argv);
diff -Nru vt-0.57721/discover.h vt-0.57721b/discover.h
--- vt-0.57721/discover.h	2021-05-14 16:42:01.000000000 +0100
+++ vt-0.57721b/discover.h	2021-05-17 14:40:07.000000000 +0100
@@ -28,7 +28,7 @@
 #include "variant_filter.h"
 #include "pileup.h"
 #include "bam_ordered_reader.h"
-#include "Rmath/Rmath.h"
+#include "Rmath.h"
 #include "log_tool.h"
 
 void discover(int argc, char ** argv);
diff -Nru vt-0.57721/estimator.h vt-0.57721b/estimator.h
--- vt-0.57721/estimator.h	2021-05-14 16:42:00.000000000 +0100
+++ vt-0.57721b/estimator.h	2021-05-17 14:40:06.000000000 +0100
@@ -27,7 +27,7 @@
 #include "utils.h"
 #include "hts_utils.h"
 #include "log_tool.h"
-#include "Rmath/Rmath.h"
+#include "Rmath.h"
 
 /**
  * Class containing estimating methods of common features.
diff -Nru vt-0.57721/hts_utils.cpp vt-0.57721b/hts_utils.cpp
--- vt-0.57721/hts_utils.cpp	2021-05-14 16:42:01.000000000 +0100
+++ vt-0.57721b/hts_utils.cpp	2021-05-17 15:11:32.000000000 +0100
@@ -22,7 +22,7 @@
 */
 
 #include "hts_utils.h"
-#include "Rmath/Rmath.h"
+#include "Rmath.h"
 
 /********
  *General
diff -Nru vt-0.57721/multiallelics_consolidator.h vt-0.57721b/multiallelics_consolidator.h
--- vt-0.57721/multiallelics_consolidator.h	2021-05-14 16:42:01.000000000 +0100
+++ vt-0.57721b/multiallelics_consolidator.h	2021-05-17 14:40:08.000000000 +0100
@@ -28,7 +28,7 @@
 #include "utils.h"
 #include "bcf_ordered_reader.h"
 #include "bcf_ordered_writer.h"
-#include "Rmath/Rmath.h"
+#include "Rmath.h"
 #include "candidate_region_extractor.h"
 #include "candidate_motif_picker.h"
 #include "flank_detector.h"
diff -Nru vt-0.57721/svm_train.h vt-0.57721b/svm_train.h
--- vt-0.57721/svm_train.h	2021-05-14 16:42:01.000000000 +0100
+++ vt-0.57721b/svm_train.h	2021-05-17 14:40:08.000000000 +0100
@@ -26,7 +26,7 @@
 
 #include "program.h"
 #include "libsvm/svm.h"
-#include "Rmath/Rmath.h"
+#include "Rmath.h"
 
 void svm_train(int argc, char ** argv);
 
diff -Nru vt-0.57721/variant_filter.h vt-0.57721b/variant_filter.h
--- vt-0.57721/variant_filter.h	2021-05-14 16:42:01.000000000 +0100
+++ vt-0.57721b/variant_filter.h	2021-05-17 14:40:07.000000000 +0100
@@ -26,7 +26,7 @@
 
 #include "hts_utils.h"
 #include "utils.h"
-#include "Rmath/Rmath.h"
+#include "Rmath.h"
 
 /**
  * Class for filtering variants in discover
diff -Nru vt-0.57721/vntr_consolidator.h vt-0.57721b/vntr_consolidator.h
--- vt-0.57721/vntr_consolidator.h	2021-05-14 16:42:01.000000000 +0100
+++ vt-0.57721b/vntr_consolidator.h	2021-05-17 14:40:08.000000000 +0100
@@ -28,7 +28,7 @@
 #include "utils.h"
 #include "bcf_ordered_reader.h"
 #include "bcf_ordered_writer.h"
-#include "Rmath/Rmath.h"
+#include "Rmath.h"
 #include "candidate_region_extractor.h"
 #include "candidate_motif_picker.h"
 #include "flank_detector.h"
diff -Nru vt-0.57721/vntr_extractor.h vt-0.57721b/vntr_extractor.h
--- vt-0.57721/vntr_extractor.h	2021-05-14 16:42:01.000000000 +0100
+++ vt-0.57721b/vntr_extractor.h	2021-05-17 14:40:07.000000000 +0100
@@ -28,7 +28,7 @@
 #include "utils.h"
 #include "bcf_ordered_reader.h"
 #include "bcf_ordered_writer.h"
-#include "Rmath/Rmath.h"
+#include "Rmath.h"
 #include "candidate_region_extractor.h"
 #include "candidate_motif_picker.h"
 #include "flank_detector.h"
